hosted_control_planes/hcp_high_availability/hcp-disaster-recovery-oadp.adoc

```diff
@@ -0,0 +1,85 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="hcp-disaster-recovery-oadp"]
+= Disaster recovery for a hosted cluster by using {oadp-short}
+include::_attributes/common-attributes.adoc[]
+:context: hcp-disaster-recovery-oadp
+
+toc::[]
+
+You can use the {oadp-short} Operator to perform disaster recovery on {aws-first} and bare metal.
+
+The disaster recovery process with {oadp-short} involves the following steps:
+
+. Preparing your platform, such as {aws-full} or bare metal to use {oadp-first}
+. Backing up the data plane workload
+. Backing up the control plane workload
+. Restoring a hosted cluster by using OADP
+
+[id="dr-oadp-prerequisites"]
```

Comment:
Per latest [repo guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-prerequisites-additional-resources-and-next-steps-titles-in-assemblies):
```suggestion
[id="prerequisites_{context}"]
```